flink-end-to-end-tests/flink-tpcds-test/tpcds-tool/query/query44.sql

```diff
@@ -1,5 +1,5 @@
 -- start query 1 in stream 0 using template ../query_templates_qualified/query44.tpl
-select  asceding.rnk, i1.i_product_name best_performing, i2.i_product_name worst_performing
+select  ascending.rnk, i1.i_product_name best_performing, i2.i_product_name worst_performing
```

Comment:
The origin TP-CDS query using `asceding`, so I would prefer not changing this file.